src/Dockerfile

```
@@ -70,8 +70,17 @@ RUN {{ spec.environment_setup }}
   {% endif %}
 {% endif %}
     INSTALL_PKGS="rsync tar gettext bind-utils nss_wrapper {{ spec.pkgs }}" && \
-{% if spec.version in ["12"] %}
-    INSTALL_PKGS="$INSTALL_PKGS rh-postgresql{{ spec.short }}-pgaudit" && \
+{% if spec.version in ["12", "13"] %}
```

```suggestion
{% if spec.version not in ["9.6", "10", "11"] %}
```

---

Maybe this could work as well considering we do not support 9.6 anymore?
```suggestion
{% if spec.version > 11 %}
```